packages/flame/test/components/shape_component_test.dart

```diff
@@ -0,0 +1,420 @@
+import 'dart:math';
+
+import 'package:flame/components.dart';
+import 'package:flame/extensions.dart';
+import 'package:flame_test/flame_test.dart';
+import 'package:test/test.dart';
+
+void main() {
+  group('ShapeComponent.containsPoint tests', () {
+    test('Simple circle contains point', () {
+      final component = CircleComponent(
+        radius: 1.0,
+        position: Vector2(1, 1),
+        anchor: Anchor.center,
+      );
+      expect(
+        component.containsPoint(Vector2.all(1.5)),
+        true,
```

Comment:
Nit: for this line and the ones below we could use isTrue and isFalse for prettier messages.

Not blocker though

Reply:
Aaah so that is what those are for, I think I removed them from some other place. 😅
Adding them now, and I will go over all the tests in a later PR where I can fix this too.

Reply:
There is a cool lint rule that can check that, maybe that is a good candidate to be added to flame lint